main.js

```diff
@@ -0,0 +1,59 @@
+console.log('hello World');
+
+//declare the id of the button
+var button = document.querySelector('#add-expense');
+var tableBody = document.querySelector('tbody');
+
+//declare var for input value from expense amount
+var inputName = document.querySelector('#expense-name');
+//declare var for input value from expense name
+var inputAmount = document.querySelector('#expense-amount');
+
+
+//*moved counter above click function, so they can be inititalized
+//create var for counter=0
+var counter;
+//create var for array of table data .amount
+var tdAmountArray;
+//use querySelectorAll to grab all the amounts on tableExpense, than:
+//create array and create for loop,
+//*move counter parseInt inside the click function*
+
+
+button.addEventListener('click', function(){
```

Comment:
Make sure you keep all the code inside these curly braces indented properly so it's easier to see where the block ends.